rust/src/rtp_parameters.rs

```diff
@@ -204,11 +204,27 @@ pub enum MediaKind {
     Video,
 }
 
+impl MediaKind {
+    pub(crate) fn to_fbs(self) -> rtp_parameters::MediaKind {
+        match self {
+            MediaKind::Audio => rtp_parameters::MediaKind::Audio,
+            MediaKind::Video => rtp_parameters::MediaKind::Video,
+        }
+    }
+
+    pub(crate) fn from_fbs(kind: rtp_parameters::MediaKind) -> Self {
+        match kind {
+            rtp_parameters::MediaKind::Audio => MediaKind::Audio,
+            rtp_parameters::MediaKind::Video => MediaKind::Video,
+        }
+    }
+}
+
 /// Error that caused [`MimeType`] parsing error.
 #[derive(Debug, Error, Eq, PartialEq)]
 pub enum ParseMimeTypeError {
-    /// Invalid input string
-    #[error("Invalid input string")]
+    /// Invalid MIME type input string
+    #[error("Invalid MIME type input string")]
```

Comment:
This helped me during debugging as I did not knw which input string it was referring to.